src/Components/About.js

```diff
@@ -0,0 +1,307 @@
+import React, { useEffect } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import aos from "aos";
+import "aos/dist/aos.css";
+import { useMediaQuery } from "react-responsive";
+
+//import pictures nightmare
+import whatsapp from "../assets/logos/whatsapp.svg";
+import calender from "../assets/logos//calendar.svg";
+import email from "../assets/logos//envelope.svg";
+import telegram from "../assets/logos//telegram.svg";
+import location from "../assets/logos//location.svg";
+import linkedIn from "../assets/logos//linkedin.svg";
+import github from "../assets/logos//github.svg";
+import facebook from "../assets/logos//facebook.svg";
+import leetcode from "../assets/logos//leetcode.svg";
```

Comment:
Maybe you could write a function that reads all files from the logos folder, and then returns them to where you need it?

Reply:
```
const getImages = () => {
  .... insert code here to read all files in logos folder ....
    return { email, telegram, location, linkedIn, ...rest }
}
```
then can use it like so
```
const { email, telegram } = getImages();